api/src/groups/test_groups.py

```diff
@@ -5,993 +5,816 @@
+    meeting = auth_client.sqla.query(Meeting).all()
+    address = auth_client.sqla.query(Address).first()
+    group = auth_client.sqla.query(Group).first()
+    # WHEN we creat a meeting with group information and address information
+    resp = auth_client.post(url_for('groups.create_meeting'),
+    json = meeting_object_factory(group_id=group.id, address_id=address.id))
+    # THEN expect the create to run OK
     assert resp.status_code == 201
 
-
 @pytest.mark.smoke
 def test_create_invalid_meeting(auth_client):
     # GIVEN an empty database
-    generate_managers(auth_client)
-    create_multiple_groups(auth_client.sqla, 1)
-    # WHEN we attempt to add invalid events
-    count = random.randint(5, 15)
-
-    for i in range(count):
-        meeting = meeting_object_factory(auth_client.sqla)
-        meeting['invalid_field'] = 4
-        resp = auth_client.post(url_for('groups.create_meeting'), json=meeting)
-
-        # THEN the response should have the correct code
-        assert resp.status_code == 422
+    # WHEN we create a invalid meeting
+    create_multiple_groups(auth_client.sqla,1)
+    create_multiple_meetings(auth_client.sqla, 3)
+    address = auth_client.sqla.query(Address).first()
+    group = auth_client.sqla.query(Group).first()
+    # WHEN we creat an invalid meeting
+    resp = auth_client.post(url_for('groups.create_meeting'),
+    json = {'active': True, 'start_time':'-20' })
+    # THEN expect error message
+    assert resp.status_code == 422
 
-    # AND the database should still be empty
-    assert auth_client.sqla.query(Meeting).count() == 0
 
 
 @pytest.mark.smoke
 def test_read_all_meetings(auth_client):
-    # Testing an empty Database
-    resp = auth_client.get(url_for('groups.read_all_meetings'))
-    assert resp.status_code == 404
-    # GIVEN a database with a number of meetings
-    count = random.randint(3, 11)
-    generate_managers(auth_client)
-    create_multiple_groups(auth_client.sqla, count)
-    create_multiple_meetings(auth_client.sqla, count)
-
-    # WHEN reading all of them
+    # GIVEN an empty database
+    # WHEN we read all meeting
+    create_multiple_groups(auth_client.sqla,1)
+    create_multiple_meetings(auth_client.sqla, 3)
+    # WHEN we read all meetings
     resp = auth_client.get(url_for('groups.read_all_meetings'))
-    # THEN we expect the correct status code
-    assert resp.status_code == 200
-
-    meetings_count = auth_client.sqla.query(Meeting).count()
-    # THEN the number of meetings should agree with the number in the database
-    assert len(resp.json) == meetings_count
-
-    # THEN each one of them should correspond to something in the database
-    for meeting in resp.json:
-        keylist = ['active', 'address_id', 'group_id', 'id']
-        filtered_meeting = {k: meeting[k] for k in meeting if k in keylist}
-        assert auth_client.sqla.query(Meeting).filter_by(**filtered_meeting).count() == 1
-
-
-def generate_addresses(auth_client, count=1):
```

Comment:
Why has this been replaced with a `pass` function?

Are these tests no longer required?